readthedocs/builds/views.py

```diff
@@ -82,6 +83,17 @@ class BuildDetail(BuildBase, DetailView):
     def get_context_data(self, **kwargs):
         context = super(BuildDetail, self).get_context_data(**kwargs)
         context['project'] = self.project
+        issue = "https://github.com/rtfd/readthedocs.org/issues/new?title="
+        title = quote("Build error with build id #%s" % self.request.path[-3:-1])
+        body = "&body=" + quote("## Details:\n\n")
+        proj = "*Project URL: https://readthedocs.org/projects/%s/\n" % self.project
+        build = "*Build URL(if applicable): https://readthedocs.org%s\n" % self.request.path
+        uname = "*Read the Docs username(if applicable): %s\n\n" % self.request.user
+        expec_res = "## Expected Result\n\n*A description of what you wanted to happen*\n\n"
+        actual_res = "## Actual Result\n\n*A description of what actually happened*"
+        body = body + quote(proj + build + uname + expec_res + actual_res)
+        issue_url = issue + title + body
+        context['issue_url'] = issue_url
```

Comment:
This is tough to follow due to excessive variable usage. Perhaps this could be simplified into a single multi line string with interpolation through `.format()` and a dictionary as input. This should feed into urlparse to actually create a URL